projects/media-viewer/src/lib/media-viewer/annotation/annotation-set.model.ts

```diff
@@ -0,0 +1,127 @@
+export interface IComment {
+  id: string;
+  annotationId: string;
+  createdBy: string;
+  createdByDetails: IdamDetails;
+  createdDate: any;
+  lastModifiedBy: string;
+  lastModifiedByDetails: IdamDetails;
+  lastModifiedDate: any;
+  content: string;
+}
+
+export class Comment implements IComment {
+  constructor(
+    public id: string,
+    public annotationId: string,
+    public createdBy: string,
+    public createdByDetails: IdamDetails,
+    public createdDate: any,
+    public lastModifiedBy: string,
+    public lastModifiedByDetails: IdamDetails,
+    public lastModifiedDate: any,
+    public content: string) {
+  }
+}
+
+export interface IRectangle {
+  id?: string;
+  annotationId?: string;
+  createdBy?: string;
+  createdByDetails?: IdamDetails;
+  createdDate?: any;
+  lastModifiedBy?: string;
+  lastModifiedByDetails?: IdamDetails;
+  lastModifiedDate?: any;
+  height?: number;
+  width?: number;
+  x?: number;
+  y?: number;
+}
+
+export class Rectangle implements IRectangle {
+  constructor(
+    public id?: string,
+    public annotationId?: string,
+    public createdBy?: string,
+    public createdByDetails?: IdamDetails,
+    public createdDate?: any,
+    public lastModifiedBy?: string,
+    public lastModifiedByDetails?: IdamDetails,
+    public lastModifiedDate?: any,
+    public height?: number,
+    public width?: number,
+    public x?: number,
+    public y?: number) {
+  }
+}
+
+export interface IAnnotation {
+  id?: string;
+  annotationSetId?: string;
+  createdBy?: string;
+  createdByDetails?: IdamDetails;
+  createdDate?: any;
+  lastModifiedBy?: string;
+  lastModifiedByDetails?: IdamDetails;
+  lastModifiedDate?: any;
+  documentId?: string;
+  page?: number;
+  color?: string;
+  comments?: Comment[];
+  rectangles?: Rectangle[];
+  type?: string;
+}
+
+export class Annotation implements IAnnotation {
+  constructor(
+    public id?: string,
+    public annotationSetId?: string,
+    public createdBy?: string,
+    public createdDate?: any,
+    public createdByDetails?: IdamDetails,
+    public lastModifiedBy?: string,
+    public lastModifiedByDetails?: IdamDetails,
+    public lastModifiedDate?: any,
+    public documentId?: string,
+    public page?: number,
```

Comment:
not sure, but this feels like it should be separated into a UI bit, which has page, color, rectangle, and the other bits to do with the user would be on a wrapper object. just a thought